src/ovirtapi.js

```diff
@@ -172,6 +172,9 @@ OvirtApi = {
       },
       cdrom: {},
       sessions: [],
+      display: {
+        smartcardEnabled: vm['display']['smartcard_enabled'] ? vm['display']['smartcard_enabled'] === 'true' : false,
```

Comment:
Is there any difference between `vm['display']['smartcard_enabled'] ? vm['display']['smartcard_enabled'] === 'true' : false` and just `vm['display']['smartcard_enabled'] === 'true'`?

Reply:
Yes, value must be `boolean` not `string`.